apps/server/README.md

```diff
@@ -36,13 +36,21 @@ docker-compose up -d
 To build the server, and all modules it depends on:
 
 ```shell
-pnpm nx build server
+pnpm nx build @overture-stack/lectern-server
+```
+or
+```shell
+pnpm build
```

Comment:
what's the difference, if any? otherwise, line 43 should suffice as "lower complexity" choice